playbook/app/pb_kits/playbook/pb_nav/_item.tsx

```diff
@@ -11,16 +9,16 @@ import Image from '../pb_image/_image'
 
 type NavItemProps = {
   active?: boolean,
-  aria?: object,
-  children: React.Node,
+  aria?: { [key: string]: string },
+  children?: React.ReactChild[] | React.ReactChild,
```

Comment:
Last time we went from React.Node to React.ReactChild, we decided to go back to React.Node | React.Node[] in case the stricter typing caused any issues fro Nitro. Can we make that change here as well?